examples/vulkan_example/imgui_impl_glfw_vulkan.cpp

```diff
@@ -236,10 +236,10 @@ void ImGui_ImplGlfwVulkan_RenderDrawLists(ImDrawData* draw_data)
         VkMappedMemoryRange range[2] = {};
         range[0].sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE;
         range[0].memory = g_VertexBufferMemory[g_FrameIndex];
-        range[0].size = vertex_size;
+        range[0].size = VK_WHOLE_SIZE;
         range[1].sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE;
         range[1].memory = g_IndexBufferMemory[g_FrameIndex];
-        range[1].size = index_size;
+        range[1].size = VK_WHOLE_SIZE;
```

Comment:
Hey!

Sorry about that. So here we are writing to mapped memory and then flushing the writes.
Vulkan requires some things about the mapped ranges, here the relevant part is: "If size is not equal to VK_WHOLE_SIZE, size must be a multiple of VkPhysicalDeviceLimits::nonCoherentAtomSize".

Since there is no check for the second condition, the validation layers now warn that the flush is not sized right (at least on my card). 

Since we were anyways flushing the whole range, I changed the size to VK_WHOLE_SIZE (meaning the full mapped range), which is legal usage (but there is no change in function).